### PR TITLE
Repair model field access in django 1.10

### PR DESCRIPTION
**Pull request by benbardin**

### CODE REVIEW

File: dynamic_rest/fields/fields.py [hunk 1]
```diff
@@ -118,8 +122,15 @@ def bind(self, *args, **kwargs):
         parent_model = getattr(self.parent.Meta, 'model', None)
 
         remote = is_field_remote(parent_model, self.source)
+
+        def get_model_field():
```
Comment (benbardin) [inline dynamic_rest/fields/fields.py]: I saw that, but there's a whole lot of logic I don't quite follow in that helper function. (See the "except" business there). Is that all correct here? I'm not sure, so I thought it safer to use this simpler logic that's identical with previous. Thoughts?

File: dynamic_rest/fields/fields.py [hunk 2]
```diff
@@ -118,8 +122,15 @@ def bind(self, *args, **kwargs):
...
+            if DJANGO110:
+                return parent_model._meta.get_field(self.source)
+            else:
+                return parent_model._meta.get_field_by_name(self.source)[0]
```
Comment (auvipy) [inline dynamic_rest/fields/fields.py]: that can be done with older releases. anyone need 1.7 will use older releases and newer release shouldn't continue to support unmaintained django versions.

File: dynamic_rest/fields/fields.py (hunk 1 above)
Comment (aleontiev) [inline dynamic_rest/fields/fields.py]: Should be fine to use that logic here; it is intended to retrieve remote fields.

File: dynamic_rest/fields/fields.py (hunk 2 above)
Comment (ryochiji) [inline dynamic_rest/fields/fields.py]: As you probably saw in #141, we need to support 1.7 for a while longer for our own purposes, but we'll likely drop support in the near future.

**Pull request merged**